### ISSUE TITLE
No data received for 9 months

### ISSUE THREAD

**Issue by LoveMyData** [planningalerts-scrapers/Bega_Valley_Applications_on_Exhibition#2]
No error.... Council system changed.

```
Auto ran revision f1bc2d21 and completed successfully 9 months ago.
run time 21 s
nothing changed in the database
7 pages scraped
```

**Comment by LoveMyData** [comment]
should be fixed by PR https://github.com/planningalerts-scrapers/Bega_Valley_Applications_on_Exhibition/pull/3

**Comment by LoveMyData** [comment]
Fixed with 5543278
